### ISSUE TITLE
Completely unusable for Fedora 31 repos

### ISSUE THREAD

**Issue by brianjmurrell** [open-sw/pkg-cacher#44]
I think this project is at it's EOL for use with Fedora.  Trying to upgrade to Fedora 31 through this proxy, it seems to constantly just stall out trying to fetch some metadata.

Sadly, about the next best thing seems to be Nexus.  But that looks to be a big(ger than necessary) Java pig.  :-(

**Comment by brianjmurrell** [comment]
Hrm.  This might have been a case of an invalid (i.e. 0 bytes) file or two in the cache.  This sort of thing really ought not to just stop pkg-cacher in it's tracks.  Invalid files should be removed and progress allowed to continue.

**Comment by ganto** [comment]
Hmn, I think the issue is the new [zchunk support](https://fedoraproject.org/wiki/Changes/Zchunk_Metadata) in the recent Fedora releases. It seems dnf now does some HTTP range requests that are not properly handled by pkg-cacher.

I found that disabling zchunk in `/etc/dnf/dnf.conf` makes it work fine with pkg-cacher again:
```
[main]
...
zchunk=False
```

**Comment by brianjmurrell** [comment]
Yes, that work-around has worked-around a number of issues with pkg-cacher.  Thanks!
